lte/protos/session_manager.proto

```diff
@@ -863,13 +921,21 @@ message M5GSMSessionContextAccess{
     repeated QosRules authorized_qos_rules = 4;
     //TS-24501 - 9.11.4.14 PDU session Aggregate Maximum Bit Rate imported from subscriberdb.proto
     AggregatedMaximumBitrate session_ambr = 5;
-    M5GSMCause M5gsm_cause = 6; //TS-24501 - 9.11.4.2
+    AmbrUnit downlink_unit_type = 6;
+    uint32   downlink_units = 7;   // Only to use lower 2 bytes (16 bit values)
+    AmbrUnit uplink_unit_type = 8;
+    uint32   uplink_units = 9;   // Only to use lower 2 bytes (16 bit values) 
+    M5GSMCause M5gsm_cause = 10; //TS-24501 - 9.11.4.2
     //PDU session is requested to be established as always-on PDU session.
-    bool always_on_pdu_session_indication = 7;
-    SscMode allowed_ssc_mode = 8; //Indicates SSC mode.
-    bool M5gsm_congetion_re_attempt_indicator = 9;
+    bool always_on_pdu_session_indication = 11;
+    SscMode allowed_ssc_mode = 12; //Indicates SSC mode.
+    bool M5gsm_congetion_re_attempt_indicator = 13;
```

Comment:
change this field name to `M5G_sm_congestion_reattempt_indicator`

Reply:
resolved